### ISSUE TITLE
[Bug] Spell checking slows down commit message entry

### ISSUE THREAD

**Issue by ravindUwU** [ForkIssues/TrackerWin#1442]
When spell checking is enabled, text entry into the commit message text boxes starts to lag. The lag seemingly increases with the length of the line on which the caret is located. #912 seems relevant.

Screencap that demonstrates the issue 👇,

https://user-images.githubusercontent.com/7352580/151988742-2d6d60db-9f03-4453-b5a5-61f0f50d6149.mp4

The [AutoHotKey](https://www.autohotkey.com/) script used to write the commit message sending keystrokes every 60ms 👇,

```autohotkey
#z::
SetKeyDelay, 60
Send, Add ESLint, Prettier, editorconfig
return
```

**Comment by DanPristupov** [comment]
Yes, I confirm. Spellchecking in WPF is very slow. That's why it's disabled by default.

**Comment by ravindUwU** [comment]
Hi Dan!  
I found an issue on the WPF repo that references this bug; have we tried the fix mentioned in the comments https://github.com/dotnet/wpf/issues/3350#issuecomment-702925249 and https://github.com/dotnet/wpf/issues/3350#issuecomment-727667670  ? 🤔
